code/game/machinery/pipe/construction.dm

```diff
@@ -125,7 +125,7 @@ Buildable meters
 	setDir(turn(dir,-90))
 	fixdir()
 
-/obj/item/pipe/wrench_act(mob/living/user, obj/item/wrench/W)
+/obj/item/pipe/wrench_act(mob/living/user, obj/item/tool)

```

Comment:
This seems to have been an oversight/mistake during conversion. wrench_act would only call if the item's tool_type was `TOOL_WRENCH`

Reply:
@ACCount12 Would you mind rolling this into your PR?